app/utils/planUtils.ts

```diff
@@ -13,11 +13,21 @@ export function isEnterprisePlan(planType: PayPlanType) {
   return planType === PayPlanType.Enterprise
 }
 
+export function isUnlimitedPlan(planType: PayPlanType) {
+  return planType === PayPlanType.PlanUnlimited
+}
+
+export function isFree(planType: PayPlanType) {
+  return planType === PayPlanType.PlanFree
+}
+
 export function isLegacyPlan(planType: PayPlanType) {
   return !(
     planType === PayPlanType.PayAsYouGoV0 ||
```

Comment:
Are we keeping these here intentionally?

If not, let's remove.

Reply:
yes to preserve any edge cases for backwards compatibility